CHANGELOG.md

```diff
@@ -6,8 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
-- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0)
+- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0).
+- Upgraded `hive-jdbc` to 2.3.4 (was 2.0.0).
+- Upgraded `hotels-oss-parent` to 4.0.1 (was 2.0.6).
+- Changed dependency from `com.microsoft.sqlserver:sqljdbc` (4.1) to `com.microsoft.sqlserver:mssql-jdbc` (6.4.0.jre7).
+- Changed dependency from `9.4.1208-jdbc42-atlassian-hosted` (4.1) to `org.postgresql:postgresql` (42.2.5.jre7).
 
+### Removed
+- DB2 and Teradata drivers (not available in Maven Central).
```

Comment:
These can't be in the pom unfortunately, the above changes will fix the build but I think the documentation and deployment procedure will need to be amended to explain how to get the drivers on the classpath when they're not open source and available in Maven Central. Going forward perhaps we shouldn't package any with Quibble and just tell people to bring their own, that can be in a different PR though.

Reply:
Agree my thinking is to perhaps to package in our own version but the open source should not have them.